lib/tasks/deployment/20190829130905_add_highlands.rake

```diff
@@ -0,0 +1,25 @@
+namespace :after_party do
+  desc 'Deployment task: add_highlands'
+  task add_highlands: :environment do
+    puts "Running deploy task 'add_highlands'"
+
+    AmrDataFeedConfig.where(
+      description: 'Highlands',
+      s3_folder: 'highlands',
+      s3_archive_folder: 'archive-highlands',
+      local_bucket_path: 'tmp/amr_files_bucket/highlands',
+      access_type: 'Email',
+      date_format: "%e %b %Y %H:%M:%S",
```

Comment:
This also gets updated as the actual records we end up inserting after the conversion have a different date format..